lib/datagrid/core.rb

```diff
@@ -56,11 +56,17 @@ def scope(&block)
           }
           self
         else
-          check_scope_defined!
-          driver.to_scope(scope_value.call)
+          scope = original_scope
+          driver.to_scope(scope)
```

Comment:
local variable with the same name as method name can be confusing:

```suggestion
          driver.to_scope(original_scope)
```

Reply:
It is not the same.

See commit with your suggestion: https://github.com/zhuravel/datagrid/commit/d5e2362c3f7e70da950260cb21e1eac044e47f73
See how it breaks build: https://github.com/zhuravel/datagrid/actions/runs/4512413662

This is due to order in which Ruby evaluates code, `original_scope` has to be called before `driver.to_scope` and not passed as an argument. These options work though:
```ruby
scope = original_scope
driver.to_scope(scope)
```

```ruby
original_scope.then do |scope|
  driver.to_scope(scope)
end
```